### PR TITLE
fix paddle.nn.loss.L1Loss OP,  add paddle.nn.functional.l1_loss OP for API2.0, test=develop

### PR DESCRIPTION
**Pull request by LutaoChu**
### PR types

New features
### PR changes

APIs
### Describe

fix paddle.nn.loss.L1Loss OP, add paddle.nn.functional.l1_loss OP for API2.0

### CODE REVIEW

File: python/paddle/nn/functional/loss.py [hunk 1]
```diff
@@ -60,3 +63,70 @@
...
+
+def l1_loss(x, label, reduction='mean', name=None):
+    """
+	:alias_main: paddle.nn.functional.l1_loss
+	:alias: paddle.nn.functional.l1_loss,paddle.nn.functional.loss.l1_loss
+    
+    This operator computes the L1 Loss of Tensor ``x`` and ``label`` as follows.
+
+    If :attr:`reduction` set to ``'none'``, the loss is:
+
```
Comment (wawltor) [inline python/paddle/nn/functional/loss.py:75]: 是否需要多一行？none mean下面都空了一行

File: python/paddle/nn/functional/loss.py [hunk 2]
```diff
@@ -60,3 +63,70 @@
...
+            "The value of 'reduction' in L1Loss should be 'sum', 'mean' or 'none', but "
+            "received %s, which is not allowed." % reduction)
+    fluid.data_feeder.check_variable_and_dtype(
+        x, 'input', ['float32', 'float64', 'int32', 'int64'], 'l1_loss')
+    fluid.data_feeder.check_variable_and_dtype(
+        label, 'label', ['float32', 'float64', 'int32', 'int64'], 'l1_loss')
+
+    if reduction == 'sum':
+        unreduced = paddle.elementwise_sub(x, label, act='abs')
+        return paddle.reduce_sum(unreduced, name=name)
```
Comment (wawltor) [inline python/paddle/nn/functional/loss.py]: 建议在2.0中将paddel.reduce_sum 改成paddel.sum paddle.reduce_mean改成paddle.mean

File: python/paddle/nn/functional/loss.py [hunk 3]
```diff
@@ -60,3 +63,70 @@
...
+    elif reduction == 'mean':
+        unreduced = paddle.elementwise_sub(x, label, act='abs')
+        return paddle.reduce_mean(unreduced, name=name)
+    else:
```
Comment (wawltor) [inline python/paddle/nn/functional/loss.py:155]: 动态图是使用core.ops

File: python/paddle/nn/functional/loss.py [hunk 4]
```diff
@@ -60,3 +63,70 @@
...
+            paddle.enable_imperative()
+            input_data = np.array([[1.5, 0.8], [0.2, 1.3]]).astype("float32")
+            label_data = np.array([[1.7, 1], [0.4, 0.5]]).astype("float32")
+            input = to_variable(input_data)
+            label = to_variable(label_data)
+            l1_loss = paddle.nn.loss.L1Loss(reduction='mean')
+            output = l1_loss(input,label)
+            print(output.numpy())  
+    """
+    if reduction not in ['sum', 'mean', 'none']:
```
Comment (wawltor) [inline python/paddle/nn/functional/loss.py:128]: 最前面是动态图的代码，使用core.ops

File: python/paddle/nn/functional/loss.py (hunk 2 above)
Comment (LutaoChu) [reply python/paddle/nn/functional/loss.py]: done

File: python/paddle/nn/functional/loss.py (hunk 1 above)
Comment (LutaoChu) [reply python/paddle/nn/functional/loss.py:75]: done

File: python/paddle/nn/functional/loss.py (hunk 2 above)
Comment (LutaoChu) [reply python/paddle/nn/functional/loss.py]: done

File: python/paddle/nn/functional/loss.py (hunk 4 above)
Comment (LutaoChu) [reply python/paddle/nn/functional/loss.py:128]: done

File: python/paddle/nn/functional/loss.py (hunk 3 above)
Comment (LutaoChu) [reply python/paddle/nn/functional/loss.py:155]: done

**Review by yaoxuefeng6 — DISMISSED** [review]
LGTM

File: python/paddle/nn/layer/loss.py [hunk 5]
```diff
@@ -250,27 +250,23 @@ def forward(self, input, label):
...
+        Out = \lvert x - label\rvert
+
+    If :attr:`reduction` set to ``'mean'``, the loss is:
+
     .. math::
-        Out = MEAN(|input - label|)
-    If :attr:`reduction` set to ``'sum'``, the reduced sum loss is:
+        Out = MEAN(\lvert x - label\rvert)
+
+    If :attr:`reduction` set to ``'sum'``, the loss is:
```
Comment (ZHUI) [inline python/paddle/nn/layer/loss.py:266]: 此处下面公式，应空一行

File: python/paddle/nn/layer/loss.py [hunk 6]
```diff
@@ -279,63 +275,53 @@ class L1Loss(fluid.dygraph.Layer):
             If :attr:`reduction` is ``'mean'``, the reduced mean loss is returned. 
             If :attr:`reduction` is ``'sum'``, the reduced sum loss is returned. 
             Default is ``'mean'``.
-    Returns:
-        A callable object of L1Loss.
```
Comment (ZHUI) [inline python/paddle/nn/layer/loss.py:283]: 各个小节之间，目测也应空一行

Comment (LutaoChu) [reply python/paddle/nn/layer/loss.py:283]: done

File: python/paddle/nn/layer/loss.py (hunk 5 above)
Comment (LutaoChu) [reply python/paddle/nn/layer/loss.py:266]: done

**Review by wawltor — APPROVED** [review]
LGTM

**Review by yaoxuefeng6 — APPROVED** [review]
LGTM

File: python/paddle/nn/layer/loss.py [hunk 7]
```diff
@@ -279,63 +276,55 @@ class L1Loss(fluid.dygraph.Layer):
             If :attr:`reduction` is ``'mean'``, the reduced mean loss is returned.
```
Comment (TCChenlong) [inline python/paddle/nn/layer/loss.py:276]: 【If :attr:`reduction` is ``'mean'``】->【If `reduction` is ``'mean'``】

**Review by TCChenlong — APPROVED** [review]
LGTM

File: python/paddle/nn/layer/loss.py (hunk 7 above)
Comment (LutaoChu) [reply python/paddle/nn/layer/loss.py:276]: 经查看，两者显示效果相同
